prophy/composite.py

```diff
@@ -295,6 +333,22 @@ def __str__(self):
                 out += field_to_string(name, tp, value)
         return out
 
+    def fields(self, prefix=''):
```

Comment:
fields or values may be valid message member names, which will clash with "fields" and "values" functions which were added